yt_dlp/__init__.py

```diff
@@ -990,6 +988,12 @@ def _real_main(argv=None):
     if opts.ffmpeg_location:
         FFmpegPostProcessor._ffmpeg_location.set(opts.ffmpeg_location)
 
+    # load all plugins into the global lookup
+    print(opts.plugin_dirs)
+    plugin_dirs.value = list(set(opts.plugin_dirs))
```

Comment:
we would lose ordering this way

Reply:
ordering doesn't matter? 

we use a set here as argparse adds duplicates for some reason

Reply:
if there are conflicting plugins that have to be loaded in a specific order i would think that having it makes more sense. Just using `orderedSet` would allow for this to be resolved by user

Reply:
I don't think we guarantee ordering atm anywhere, so I don't think this is something we need to support/test right now

Reply:
thats fair, if it ever comes up we cal always go back

Reply:
still, we might as well use `orderedSet`, no?